drivers/i2c/i2c_npcx_port.c

```diff
@@ -143,6 +143,42 @@ static int i2c_npcx_port_recover_bus(const struct device *dev)
 	return ret;
 }
 
+#ifdef CONFIG_I2C_TARGET
+static int i2c_npcx_target_register(const struct device *dev,
+				  struct i2c_target_config *target_cfg)
+{
+	const struct i2c_npcx_port_config *const config = dev->config;
+
+	if (!target_cfg) {
+		return -EINVAL;
+	}
+
+	if (config->i2c_ctrl == NULL) {
+		LOG_ERR("Cannot find i2c controller on port%02x!",
+								config->port);
+		return -EIO;
+	}
+
+	/* Register i2c slave target */
```

Comment:
nit: I'd drop the comment (and the one on unregister), it literally repeats the function name

Reply:
Done.